### PR TITLE
Finalize Copy for Production

### PR DESCRIPTION
**Pull request by hhff**

### CODE REVIEW

**Comment by hhff** [comment]
Please double check for typos @chrismekim !

File: src/index.html [hunk 1]
```diff
@@ -256,12 +259,12 @@ <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">It just takes a few steps
       </section>
 
       <section class="smaller-content-container inner-content-padding-x mxauto flex-col fade-in-up">
-        <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">Expecting more from ourselves and helping inspire our clients.</h3>
+        <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">Expecting more from ourselves, and helping inspire our clients.</h3>
         <p class="off-1 md:off-4 md:col-8 xl:off-6 xl:col-6 text-paragraph mb2_5 atlas">
-          We live and work in New York City, where a new DTC brand is released on what feels like a daily basis. Sanctuary is now after four years in the industry in a fortunate position to be able to say no to potential clients whose mission we don't align with. We're fortunate to work with incredible clients who are moving. 
+          We live and work in New York City, where a new DTC brand is released on what feels like a daily basis. After four years in this industry, we're now in a fortunate position to be able to say turn down potential clients whose mission don't align with ours.
```
Comment (chrismekim) [inline src/index.html:264]: to be able to say turn down potential clients whose mission
- remove say, make mission plural

File: src/offsetting.html [hunk 2]
```diff
@@ -99,15 +99,35 @@ <h1 class="md:off-6 xl:off-4 md:justify-end lg:justify-start flex text-title ser
...
+          <div class="flex flex-row space-between pb2 md:pb3">
+            <span class="list-number">(1.)</span>
+            <p>
+              Transparency — We expect that any offset we purchase can prove without doubt and in the open that an individual credit can not be "double counted", or purchased more than once.
+            </p>
+          </div>
+          <div class="flex flex-row space-between pb2 md:pb3">
+            <span class="list-number">(2.)</span>
+            <p>
+              Permanence — We do not buy credits when there is not a strong guarentee that the unit or drawdown will not be permanent, or is liable to be undone.
```
Comment (chrismekim) [inline src/offsetting.html:115]: guarantee

File: src/offsetting.html [hunk 3]
```diff
@@ -99,15 +99,35 @@ <h1 class="md:off-6 xl:off-4 md:justify-end lg:justify-start flex text-title ser
...
+            </p>
+          </div>
+          <div class="flex flex-row space-between pb2 md:pb3">
+            <span class="list-number">(3.)</span>
+            <p>
+              Additionality — We only buy credits when the supplier can clearly prove that buy facilitating this activity, more carbon was drawn down than would have been.
```
Comment (chrismekim) [inline src/offsetting.html:121]: buy -> by

File: src/offsetting.html [hunk 4]
```diff
@@ -188,7 +211,7 @@ <h3 class="text-subtitle serif mb3 lg:col-6">Ethical pricing for renwable energy
       </section>
 
       <section class="smaller-content-container inner-content-padding-x mxauto flex-col fade-in-up">
-        <h3 class="text-subtitle serif mb3 lg:col-5">Premature technologies are a gamble and not a best use of our resources.</h3>
+        <h3 class="text-subtitle serif mb3 lg:col-5">Early stage technologies may not a best use of our resources (yet).</h3>
```
Comment (chrismekim) [inline src/offsetting.html:214]: may not **be**?

File: src/emissions.html [hunk 5]
```diff
@@ -395,12 +395,12 @@ <h3 class="text-subtitle serif mb3 md:col-9 xl:col-6">Calculating our carbon foo
...
-          Going to all of this effort without preparing a clear plan to reduce emissions overall is a pointless endeavor. The long term health of our planet requires a reduction in emissions, which can't be achieved if we just meet the status quo each year.
+          Going to all of this effort without preparing a clear plan to reduce emissions overall is a pointless endeavor. The long term health of our planet requires a reduction in emissions, which can't be achieved if we simply meet the status quo each year.
         </p>
         <p class="off-1 md:off-4 md:col-8 xl:off-6 xl:col-6 text-paragraph atlas md:col-6">
-          When performing these studies, we assessed each category, tried to see if we could do better. We were surprised! We found a bunch of places we could lower our emissions simply by choosing a different provider, or doing a little leg work to set up better systems. 
+          When performing these studies, we assessed each category, tried to see if we could do better. We were surprised! We found a bunch of places we could lower our emissions simply by choosing a different provider, or doing a little leg work to set up better systems.
```
Comment (chrismekim) [inline src/emissions.html:403]: ...we assessed each category, **and** tried to see if we could do better

**Pull request merged**